src/router/index.ts

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue';
 import Router from 'vue-router';
-import Hello from '@/pages/Hello';
+import Hello from '@/views/pages/Hello.vue';
```

Comment:
Have to add `.vue` to vue component imports for the time being to get full editor support